pkg/operator/ceph/version/version.go

```diff
@@ -50,7 +50,7 @@ var (
 	Tentacle = CephVersion{20, 0, 0, 0, ""}
 
```

Comment:
Let's define Urchin as the next unsupported, and also see #14454 for a few more examples to copy for how Tentacle was added.
```suggestion
Urchin = CephVersion{21, 0, 0, 0, ""}
```